.github/workflows/main.yml

```diff
@@ -0,0 +1,16 @@
+name: test_sphinx_build
+on: [push, pull_request]
+
+jobs:
+  build_docs_job:
+    runs-on: ubuntu-22.04
```

Comment:
This PR looks good to me.
Just out of curiosity, are we freezing the build only on this specific version of ubuntu, or `ubuntu-latest` could be an alternative, for a longer run into the future?

Reply:
it's better to keep this fixed, so if the build fails we know it's not due to the ubuntu version used.
22.04 is a long time release, so we're good for a few more years (can always be updated later of course).